models/public/mixnet-l/README.md

```diff
@@ -0,0 +1,73 @@
+# mixnet-l
+
+## Use Case and High-Level Description
+MixNets are a family of mobile-sizes image classification models equipped with MixConv,
+a new type of mixed depthwise convolutions. There are three MixNet architectures -
+`MixNet-S` (Small), `MixNet-M` (Middle), `MixNet-L` (Large). The main differences are using
+MixConv with different kernel sizes and number of layers. Using `MixNet-L` allows to achieve greater accuracy.
+Use this [link](https://arxiv.org/abs/1907.09595) to learn more about MixNet architectures.
+`MixNet-L` was pretrained in TensorFlow\*.
+All the MixNet models have been pretrained on the ImageNet\* image database.
+For details about this family of models, check out the [TensorFlow Cloud TPU repository](https://github.com/tensorflow/tpu/tree/master/models/official/mnasnet/mixnet).
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 0.565         |
+| MParams           | 7.300         |
+| Source framework  | TensorFlow\*  |
+
+## Accuracy
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| Top 1  | 78.30%         | 78.30%          |
+| Top 5  | 93.91%         | 93.91%          |
+
+## Input
+
+### Original Model
+
+Image, name - `image`,  shape - `[1x224x224x3]`, format is `[BxHxWxC]`, where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `RGB`.
+
+### Converted Model
+
+Image, name - `IteratorGetNext/placeholder_out_port_0`,  shape - `[1x3x224x224]`, format is `[BxCxHxW]`, where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Output
+
+### Original Model
+
+Object classifier according to ImageNet classes, name - `logits`,  shape - `1,1000`, output data format is `B,C` where:
+
+- `B` - batch size
+- `C` - predicted logits for each class
+
+### Converted Model
+
+Object classifier according to ImageNet classes, name - `logits`,  shape - `1,1000`, output data format is `B,C` where:
+
+- `B` - batch size
+- `C` - predicted logits for each class
+
```

Comment:
add new common to all models section, before Legal Information

`
## Download a Model and Convert it into Inference Engine Format

You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.

An example of using the Model Downloader:
```
python3 <omz_dir>/tools/downloader/downloader.py --name <model_name>
```

An example of using the Model Converter:
```
python3 <omz_dir>/tools/downloader/converter.py --name <model_name>
```
`

Reply:
added